app/assets/stylesheets/components/_highlight-boxes.scss

```diff
@@ -11,7 +11,7 @@
   width: 33%;
```

Comment:
The code seems to be outside the scope of this PR so again this is only a mention rather than necessary for investigation, but due to the irregular padding on the LI element the boxes overall don't fill the full width of the space available, see below (I've cut out some of the elements on the page so it's more obvious when compared with the 'Guidance and Regulation' title). This could be solved using flexbox's justify-content: space-between option and removing the padding altogether.

![screen shot 2018-04-12 at 13 22 37](https://user-images.githubusercontent.com/861310/38677052-ffe3917c-3e54-11e8-8ab7-39e1bba06789.png)